From 4ebd82dd358f038a27f450936f4f4f7d1c668f1c Mon Sep 17 00:00:00 2001
Message-ID: <4ebd82dd358f038a27f450936f4f4f7d1c668f1c.1720090727.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1720090727.git.sharlatanus@gmail.com>
References: <cover.1720090727.git.sharlatanus@gmail.com>
From: Rodion Goritskov <rodion.goritskov@gmail.com>
Date: Wed, 3 Jul 2024 23:22:31 +0100
Subject: [PATCH v2 4/9] gnu: Add go-github-com-abadojack-whatlanggo.

* gnu/packages/golang-xyz.scm (go-github-com-abadojack-whatlanggo): New variable.

Signed-off-by: Sharlatan Hellseher <sharlatanus@gmail.com>
Change-Id: I49a0d5c9d97ba9eb1279b75f5177966e800a2475
---
 gnu/packages/golang-xyz.scm | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/gnu/packages/golang-xyz.scm b/gnu/packages/golang-xyz.scm
index 54b9c45ed6..d63c184bf6 100644
--- a/gnu/packages/golang-xyz.scm
+++ b/gnu/packages/golang-xyz.scm
@@ -373,6 +373,31 @@ (define-public go-github-com-a8m-envsubst
 substitution.")
     (license license:expat)))
 
+(define-public go-github-com-abadojack-whatlanggo
+  (package
+    (name "go-github-com-abadojack-whatlanggo")
+    (version "1.0.1")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/abadojack/whatlanggo")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1pidd5dqvcnqjjka12h0clj3mmq0j3bpanf9153schsx85xz7mzx"))))
+    (build-system go-build-system)
+    (arguments
+     (list
+      #:import-path "github.com/abadojack/whatlanggo"))
+    (home-page "https://github.com/abadojack/whatlanggo")
+    (synopsis "Natural language detection library for Go")
+    (description
+     "Package detects natural languages and scripts (writing systems).
+Languages are represented by a determined list of constants while scripts are
+represented by RangeTable.")
+    (license license:expat)))
+
 (define-public go-github-com-adrg-strutil
   (package
     (name "go-github-com-adrg-strutil")
-- 
2.41.0

